python/lsst/meas/base/baseMeasurement.py

```diff
@@ -30,17 +30,17 @@
 from .applyApCorr import ApplyApCorrTask
 from .pluginRegistry import PluginMap
 from .baseLib import FatalAlgorithmError, MeasurementError
+from .pluginsBase import BasePluginConfig, BasePlugin
 from .noiseReplacer import NoiseReplacerConfig
-from .transforms import PassThroughTransform
 
-__all__ = ("BasePluginConfig", "BasePlugin", "BaseMeasurementConfig", "BaseMeasurementTask")
+__all__ = ("BaseMeasurementPluginConfig", "BaseMeasurementPlugin", "BaseMeasurementConfig", "BaseMeasurementTask")
 
 # Exceptions that the measurement tasks should always propagate up to their callers
 FATAL_EXCEPTIONS = (MemoryError, FatalAlgorithmError)
 
-class BasePluginConfig(lsst.pex.config.Config):
-    """!
-    Base class measurement Plugin config classes.
+class BaseMeasurementPluginConfig(BasePluginConfig):
+    """
+    Base config class for all measurement plugins
```

Comment:
If you're going to strip the bang (and I don't know if it's required when there are no doxygen markups), then move the next line up to the classic summary line.